azkaban-common/src/main/java/azkaban/container/models/AzKubernetesV1SpecBuilder.java

```diff
@@ -160,7 +161,21 @@ public AzKubernetesV1SpecBuilder addInitContainerType(String name, String image,
         LOGGER.debug("Added init container to the pod spec");
         return this;
     }
-
+    public AzKubernetesV1SpecBuilder addInitContainerType(String image, ImagePullPolicy imagePullPolicy,
```

Comment:
The function name looks a little bit weird to me.
It is only used for "security-init" but named as "addInitContainerType" (general name) & didn't check if initContainerType == InitContainerType.SECURITY.

How about renaming it to addSecurityInitContainer and check the initContainerType == InitContainerType.SECURITY inside of this function?